docs/releases/2.0.txt

```diff
@@ -219,6 +219,9 @@ Models
   from the database. For databases that don't support server-side cursors, it
   controls the number of results Django fetches from the database adapter.
 
+* Added the ``db_tablespace`` named argument to the
```

Comment:
Added the db_tablespace (link to .. attribute:: Index.db_tablespace) parameter to class-based indexes.